website/src/components/UserTable.tsx

```diff
@@ -47,17 +50,23 @@ const columns: DataTableColumnDef<User>[] = [
 export const UserTable = memo(function UserTable() {
   const { pagination, resetCursor, toNextPage, toPreviousPage } = useCursorPagination();
   const [filterValues, setFilterValues] = useState<FilterItem[]>([]);
-  const handleFilterValuesChange = (values: FilterItem[]) => {
+  const [filterOption, setFilterOption] = useState<string>("display_name");
```

Comment:
You dont need this state. The last FilterItem is the column you want, so just discard the rest values and keep the last one only

Reply:
I went and removed the state and used the last FilterItem